base/applications/mspaint/precomp.h

```diff
@@ -43,5 +42,6 @@
 #include "toolsettings.h"
 #include "toolsmodel.h"
 #include "winproc.h"
+#include "common.h"
```

Comment:
What was the exact reason you had to move this include?

Reply:
Nothing.

Reply:
To use the global variable, its type should be predefined.

Reply:
https://github.com/reactos/reactos/pull/4188#discussion_r775185392